client/src/plugins/error-tracking/__tests__/ErrorTrackingSpec.js

```diff
@@ -357,8 +360,85 @@ describe('<ErrorTracking>', () => {
       { key: 'plugins', value: plugins.map(({ name }) => name) }
     );
   });
+
+
+  describe('Path normalization', () => {
+
+    it('should normalize Windows paths', () => {
+
+      return normalizationTest('/C:/Users/user/test-user/Desktop/Camunda/resources/app.asar/public/');
+    });
+
+
+    it('should normalize Linux paths', () => {
+
+      return normalizationTest('/home/testuser/Aplications/camunda-modeler-4.0.0-linux-x64/resources/app.asar/public/');
+    });
+
+
+    it('should normalize Mac paths', () => {
+
+      return normalizationTest('/Applications/Camunda Modeler.app/Contents/Resources/app.asar/public/');
+    });
+
+
+    it('should normalize dev paths', () => {
+
+      return normalizationTest('webpack-internal:///./src/plugins/camunda-plugin/deployment-tool/');
+    });
+  });
 });
 
+async function normalizationTest(prefix) {
```

Comment:
maybe we can rename it to `expectNormalization`?

Reply:
Good idea

Reply:
Done via https://github.com/camunda/camunda-modeler/pull/1846/commits/dc92106c021ec390281d9d1216f23b3ee0790aa2